red5pro-installer/r5p_config_node_apps_plugins.sh

```diff
@@ -230,6 +230,7 @@ config_node_apps_plugins(){
 
         log_i "Configuration Live App red5-web.properties with MOCK Round trip server ..."
         {
+            echo ""
```

Comment:
Hi @iolesyk Do we require this extra spaces in `/usr/local/red5pro/webapps/live/WEB-INF/red5-web.properties` ?

Reply:
Yes, this is fix issue related with inserting variable in the existing commented line in configuration file.

Reply:
Hi @iolesyk Thanks for your feedback, I think we can merge this to `master` branch.